mods/cnc/maps/nod06a/nod06a.lua

```diff
@@ -167,8 +153,6 @@ WorldLoaded = function()
 
 	OnAnyDamaged(Atk2ActorTriggerActivator, Atk2TriggerFunction)
 
-	Trigger.OnDamaged(Atk3Activator, Atk3TriggerFunction)
```

Comment:
```lua
if Map.LobbyOption("difficulty") == "though" then
    Trigger.OnDamaged(Atk3Activator, Atk3TriggerFunction)
end
```
then you have to add in the [`rules.yaml`](https://github.com/OpenRA/OpenRA/blob/bleed/mods/cnc/maps/nod06a/rules.yaml#L11):
```yaml
	ScriptLobbyDropdown@difficulty:
		ID: difficulty
		Label: Difficulty
		Values:
			normal: Normal
			tough: Real tough guy
		Default: normal
```

Reply:
Thanks!